tests/sentry/integrations/test_client.py

```diff
@@ -142,6 +153,106 @@ def test_default_redirect_behaviour(self):
         resp = ApiClient().delete("http://example.com/3", allow_redirects=True)
         assert resp.status_code == destination_status
 
+    def test_ConnectionError_handling(self):
```

Comment:
```suggestion
    def test_connection_error_handling(self):
```

I believe we try to follow snake case which does not use upper case.